content/20.zksync-protocol/10.api/40.zks-rpc.md

```diff
@@ -83,40 +79,57 @@ curl --request POST \
   --data '{
       "jsonrpc": "2.0",
       "id": 2,
-      "method": "zks_estimateGasL1ToL2",
-      "params": [
-        {
-          "from": "0x1111111111111111111111111111111111111111",
-          "to": "0x2222222222222222222222222222222222222222",
-          "data": "0xffffffff"
-        }
-      ]
-    }'
+      "method": "zks_gasPerPubdata",
+      "params": []
+  }'
 ```
 
 #### Example Response
 
-```json
-{
-  "jsonrpc": "2.0",
-  "result": "0x25f64db",
-  "id": 2
-}
+```sh
+
```

Comment:
And the excerpt from the previous comment should go here instead

Reply:
fixed